test/functional/gridfs_stream_tests.js

```diff
@@ -1186,29 +1186,31 @@ describe('GridFS Stream', function() {
                   client.close();
                   done();
                 });
+                //temporary fix to bug in test.
```

Comment:
Is there a ticket to fix this test? If so, can you add the ticket number to this comment and make it a TODO? i.e. `TODO: fix this test for NODE-xxxx`

Reply:
added a comment referring to NODE-2096! https://jira.mongodb.org/browse/NODE-2096